plugins/woocommerce/tests/php/src/Internal/Admin/Suggestions/PaymentExtensionSuggestionsTest.php

```diff
@@ -392,7 +392,7 @@ public function data_provider_get_country_extensions_count_with_merchant_selling
 			'RO' => 7,
 			'SM' => 2,
 			'SK' => 6,
-			'ES' => 10,
+			'ES' => 11, // Including Square (in-person).
```

Comment:
```suggestion
			'ES' => 11,
```
Not sure that we need the extra comment

Reply:
changed in [7002365](https://github.com/woocommerce/woocommerce/pull/55363/commits/7002365faeae7d9f976575d44f2399976ec2f621) . thanks